serverless-gcp-typescript/app/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "serverless-gcp-nodejs",
+    "name": "${PROJECT}",
```

Comment:
Actually -- this one should be omitted (since it belongs to the container app)! I'll remove.